app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -250,11 +250,11 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
   }
 
   fun showBloodPressureEntrySheet(patientUuid: UUID) {
-    activity.startActivityForResult(BloodPressureEntrySheet.intent(context, patientUuid), REQCODE_BP_SAVED)
+    activity.startActivityForResult(BloodPressureEntrySheet.intentForNewBp(context, patientUuid), REQCODE_BP_SAVED)
   }
 
   fun showBloodPressureUpdateSheet(bloodPressureMeasurementUuid: UUID) {
-
+    activity.startActivityForResult(BloodPressureEntrySheet.intentForUpdateBp(context, bloodPressureMeasurementUuid), REQCODE_BP_SAVED)
```

Comment:
Can I ask you to break this statement down? A lot is happening here? Same for `showBloodPressureEntrySheet()`

Something along the lines of,

```kotlin
val intent = ...
activity.startActivityForResult(...)
```

Reply:
and maybe also `showScheduleAppointmentSheet()`?

Reply:
Done.